compose/backend.yml

```
@@ -75,6 +75,9 @@ services:
       DATA_STORE_DB_PASSWORD: "${DB_PASSWORD}"
       DATA_STORE_DB_USE_SSL: "${DB_USE_SSL}"
       DATA_STORE_IDAM_KEY: "${IDAM_KEY_CCD_DATA_STORE}"
+      REDIS_PORT: "${REDIS_PORT:-6379}"
+      REDIS_HOST: "${REDIS_HOST:-host.docker.internal}"
```

I think we should add `ccd-redis` into the `ccd-network`, and then this becomes:
```suggestion
      REDIS_HOST: "${REDIS_HOST:-ccd-redis}"
```